src/components/side-panel/layer-panel/color-selector.js

```diff
@@ -85,7 +88,7 @@ class ColorSelector extends Component {
   };
 
   handleClickOutside = e => {
-    if (this.state.editing !== false) {
+    if (this.state.editing !== false & this.props.showSketcher===false) {
```

Comment:
> Seeing this error
> <img alt="Screen Shot 2019-07-09 at 4 46 12 PM" width="802" src="https://user-images.githubusercontent.com/3605556/60930131-1a6b0d80-a269-11e9-8c15-7ef007e2709d.png">
> 
> To reproduce:
> 
> 1. load Unemployment for US demo map: http://localhost:8080/demo/county_unemployment
> 2. Click on polygon layer
> 3. click on fill color color selector
> 4. click on custom toggle

this was fixed now

Reply:
> `&&`
> This can just if `(this.state.editing && this.props.showSketcher)`

meant the opposite though: showSketcher need to be falsy. Could shorten the original this.state.editing if needed

Reply:
had to revert to this.state.editing !== false, otherwise onClickoutside doesn't work. this is because editing state takes nonbinary index in _showDropdown function. The rest resolved.